src/components/Summary.tsx

```diff
@@ -64,6 +65,25 @@ export default class Summary extends React.Component<SummaryProps, SummaryState>
     let client: Client = this.context.client;
     // @ts-ignore
     const patient = this.context.patient;
+
+    // @ts-ignore
+    const practitioner = this.context.practitioner;
+    if (practitioner) {
+      if (!patient.generalPractitioner) patient.generalPractitioner = [];
+      if (
+        !patient.generalPractitioner.find(
+          (p:Practitioner) => p.reference && p.reference.split("/")[1] === practitioner.id
+        )
+      ) {
+        // add current user to be one of the patient's general practitioners (list of followers)
+        const practitionerReference = {
+          type: "Practitioner",
+          reference: `Practitioner/${practitioner.id}`,
+        };
+        patient.generalPractitioner.push(practitionerReference);
```

Comment:
@achen2401  Does this align w/ https://www.pivotaltracker.com/story/show/185407795/comments/237236073 ?

Reply:
@mcjustin yes, noting that I haven't implemented the part where all the patient's followers are part of the careplan's `careteam`